### PR TITLE
[PDR-435] Add winter cope survey to PDR.

### PR DESCRIPTION
**Pull request by robabram**
## Description of changes/additions
Add support for winter COPE survey for PDR.

## Tests
- [x] unit tests

### CODE REVIEW

File: rdr_service/model/bq_questionnaires.py [hunk 1]
```diff
@@ -910,6 +910,80 @@ class BQPDRSDOHView(BQModuleView):
     _show_created = True
 
 
+class BQPDRCOPEVaccine3Schema(_BQModuleSchema):
+    """ COPE Vaccine Survey 3 (Winter 2021) """
+    _module = 'cope_vaccine3'
+    _force_boolean_fields = (
```
Comment (dtharpeuno) [inline rdr_service/model/bq_questionnaires.py:916]: Sheesh...

**Review by dtharpeuno — APPROVED** [review]

File: rdr_service/model/bq_questionnaires.py (hunk 1 above)
Comment (robabram) [reply rdr_service/model/bq_questionnaires.py:916]: There are 6 or 7 distinct covid surveys, their code id naming hasn't been very consistent.

File: rdr_service/model/bq_questionnaires.py [hunk 2]
```diff
@@ -910,6 +910,80 @@ class BQPDRSDOHView(BQModuleView):
...
+        'cdc_covid_xx_b_firstdose_other',
```
Comment (pegbertsch) [inline rdr_service/model/bq_questionnaires.py:917]: You have more visibility than I do into the codebook extract, but are all of these codes (say, dose5 fields and up) expected to have response values in this next Winter Survey?   Or are they trying to create a generic set of codes codebook will also apply to future COPE Vaccine modules should there also be additional boosters?

Do we know which codes the dashboard team is expecting to extract into the `v_pdr_mod_cope_tableau` view?  would have to update all the SELECTs that are part of the UNION ALL logic

**Review by pegbertsch — COMMENTED** [review]
Do you need to add the new module to the model/__init__.py file?

**Comment by robabram** [comment]
Yes, good catch.

File: rdr_service/model/bq_questionnaires.py (hunk 2 above)
Comment (robabram) [reply rdr_service/model/bq_questionnaires.py:917]: I don't have visibility in to that.

File: rdr_service/model/bq_questionnaires.py (hunk 1 above)
Comment (pegbertsch) [reply rdr_service/model/bq_questionnaires.py:916]: There are some shared "forced boolean" fields between all the surveys that included vaccine questions (Feb Cope + the minute surveys?)

But I don't understand why this survey has questions about so many doses.   (Haven't seen the codebook)

File: rdr_service/model/bq_questionnaires.py (hunk 2 above)
Comment (pegbertsch) [reply rdr_service/model/bq_questionnaires.py:917]: Can you tell what the display values are for all those question codes, or do we have to wait until Kenny has run his import to look at the display strings in RDR?

**Review by pegbertsch — APPROVED** [review]

**Pull request merged**